clang/test/dpct/cub/warplevel/shuffle.cu

```diff
@@ -64,14 +64,36 @@ __global__ void ShuffleIndexKernel3(int* data) {
   int threadid = threadIdx.x;
   int input = data[threadid];
   int output = 0;
-// CHECK: /*
-// CHECK-NEXT: DPCT1007:{{[0-9]+}}: Migration of cub::ShuffleUp is not supported.
-// CHECK-NEXT: */
-// CHECK-NEXT: output = cub::ShuffleUp<32>(input, 0, 0, 0xaaaaaaaa);
+  // CHECK: /*
+  // CHECK-NEXT: DPCT1108:{{.*}}: 'cub::ShuffleUp' was migrated with the experimental feature masked sub_group function which may not be supported by all compilers or runtimes. You may need to adjust the code.
```

Comment:
the test should enable the option: --use-experimental-features= non-uniform-groups ?

Reply:
Done